tox.ini

```
@@ -152,7 +150,7 @@ usefixtures =
 [testenv]
 deps =
     # Base Dependencies
-    {py36,py37,py38,py39,py310,pypy36}: pytest==6.2.5
+    {py37,py38,py39,py310,pypy36}: pytest==6.2.5
```

```suggestion
    {py37,py38,py39,py310,pypy36,pypy37}: pytest==6.2.5
```

We must have missed this in a different PR, let's change it here.